homeassistant/components/epson/config_flow.py

```diff
@@ -15,7 +14,14 @@
     {
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_NAME, default=DOMAIN): str,
-        vol.Optional(CONF_PORT): cv.port,
+        vol.Optional(CONF_PORT): int,
```

Comment:
We can remove the `DATA_SCHEMA` constant since it's not used anymore.

Reply:
I mixed up legacy schema with config_flow schema, I will fix it asap